client/app/initializers/promise.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function initialize() {
+  window.Promise = Ember.RSVP.Promise;
```

Comment:
Why does `Ember.RSVP.Promise` need to go in an initializer if it is already accessible by importing Ember`?